mardor/marfile.py

```diff
@@ -129,16 +129,20 @@ class MarFile:
 
     # TODO: Handle writing the product information block
 
-    def __init__(self, name, mode="r", signature_versions=[]):
+    def __init__(self, name, fileobj=None, mode="r", signature_versions=[]):
```

Comment:
would it make sense to move fileobj after mode?

when opening by name, you're more likely to want to do MarFile("filename", "r")